workbench-core/example/infrastructure/integration-tests/tests/authentication/middleware.test.ts

```diff
@@ -43,4 +43,70 @@ describe('authentication middleware integration tests', () => {
       );
     });
   });
+
+  describe('csurf', () => {
+    it('should throw 401 when the user does not provide a csrf-token header', async () => {
```

Comment:
is this route exclusively for fake variables? can it include valid csrf tokens?

Reply:
It handles valid and invalid csrf tokens. I just used a dummy route to keep this test isolated. It will return a 403 on a "success" (ie it made it through the csrf protection step and is getting stopped by authZ)